<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenPrincipSC</class>
 <widget class="QMainWindow" name="FenPrincipSC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FenPrincipSC</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Fond de la fenêtre principale */
QMainWindow {
    background-color: #f5f7fa;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Titre &quot;Suivi des candidatures&quot; */
QLabel#Title {
    font-size: 28px;
    font-weight: bold;
    color: #2c3e50;
    margin-bottom: 20px;
}
</string>
    </property>
    <property name="text">
     <string>Suivi des candidatures</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Statistiques">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>531</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Conteneur des statistiques */
QGroupBox#Statistiques {
    border: 2px solid #e0e0e0;
    border-radius: 8px;
    background-color: white;
    padding: 15px;
    margin-bottom: 20px;
    font-size: 16px;
    font-weight: bold;
    color: #2c3e50;
}

/* Libellés des statistiques */
QLabel#Lbl_TtC,
QLabel#Lbl_TtCA,
QLabel#Lbl_TtCR,
QLabel#Lbl_TtEA {
    font-size: 14px;
    color: #7f8c8d;
    font-weight: normal;
}

/* Valeurs numériques des statistiques */
QLabel#Lbl_NumTtC {
    font-size: 24px;
    font-weight: bold;
    color: #3498db;
}

QLabel#Lbl_NumTtCA {
    font-size: 24px;
    font-weight: bold;
    color: #27ae60;
}

QLabel#Lbl_NumTtCR {
    font-size: 24px;
    font-weight: bold;
    color: #e74c3c;
}

QLabel#Lbl_NumTtEA {
    font-size: 24px;
    font-weight: bold;
    color: #f39c12;
}
</string>
    </property>
    <property name="title">
     <string>Données</string>
    </property>
    <widget class="QLabel" name="Lbl_TtC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Total candidatures :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_TtCA">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Candidatures acceptées :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_TtCR">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Candidatures refusées :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_TtEA">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>75</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>En attente :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_NumTtC">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_NumTtCA">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>35</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_NumTtCR">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_NumTtEA">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="TableauCandid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>1071</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Tableau des candidatures */
QTableWidget#TableauCandid {
    background-color: white;
    border: 1px solid #e0e0e0;
    border-radius: 6px;
    gridline-color: #e0e0e0;
    font-size: 13px;
}

/* En-têtes du tableau */
QTableWidget#TableauCandid::item {
    padding: 6px 10px;
    color: #2c3e50;
	line-height: 120%;
}

QHeaderView::section {
    background-color: #f8f9fa;
    padding: 8px;
    border: 1px solid #e0e0e0;
    font-weight: bold;
    color: #2c3e50;
}

/* Lignes paires/impaires pour un effet &quot;zébré&quot; */
QTableWidget#TableauCandid QTableWidgetItem {
    padding: 6px 10px;
}

QTableWidget#TableauCandid QTableWidgetItem::item:selected {
    background-color: #bbdefb;
}

/* Lignes paires/impaires (effet zébré) */
QTableWidget#TableauCandid::item:alternate {
    background-color: #f8f9fa;
}

/* Ligne survolée */
QTableWidget#TableauCandid::item:hover {
    background-color: #edf6ff;
}

/* Ligne sélectionnée */
QTableWidget#TableauCandid::item:selected {
    background-color: #bbdefb;
    color: #000;
}</string>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::PenStyle::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Nom Ent.</string>
     </property>
     <property name="toolTip">
      <string>Nom de l'entreprise</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Info Ent.</string>
     </property>
     <property name="toolTip">
      <string>Info sur l'entreprise</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Poste</string>
     </property>
     <property name="toolTip">
      <string>Poste recherché</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lien Candidature</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Téléphone</string>
     </property>
     <property name="toolTip">
      <string>Téléphone de l'entreprise</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E-Mail</string>
     </property>
     <property name="toolTip">
      <string>Email de l'Entreprise</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Adresse</string>
     </property>
     <property name="toolTip">
      <string>Adresse Postal de l'Entreprise</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date Envoie CV</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date Réponse</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Etat Réponse</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="QPB_AddEnt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Bouton &quot;Ajouter une entreprise&quot; */
QPushButton#QPB_AddEnt {
    background-color: #3498db;
    color: white;
    border: none;
    padding: 8px 16px;
    font-size: 14px;
    border-radius: 4px;
}

QPushButton#QPB_AddEnt:hover {
    background-color: #2980b9;
}
</string>
    </property>
    <property name="text">
     <string>Ajouter une entreprise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QPB_Save">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>53</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Boutons Sauvegarder et Charger */
QPushButton#QPB_Save {
    background-color: #27ae60; /* vert moderne */
    color: white;
    border: none;
    padding: 10px 20px;
    font-size: 14px;
    font-weight: bold;
    border-radius: 6px;
}

QPushButton#QPB_Save:hover {
    background-color: #1e8449;
}

QPushButton#QPB_Save:pressed {
    background-color: #145a32;
}

QPushButton#QPB_Save:disabled {
    background-color: #95a5a6;
    color: #ecf0f1;
}
</string>
    </property>
    <property name="text">
     <string>💾 Sauvegarder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QPB_Load">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Boutons Sauvegarder et Charger */
QPushButton#QPB_Load {
    background-color: #3498db;
    color: white;
    border: none;
    padding: 10px 20px;
    font-size: 14px;
    font-weight: bold;
    border-radius: 6px;
}

QPushButton#QPB_Load:hover {
    background-color: #2980b9;
}

QPushButton#QPB_Load:pressed {
    background-color: #1c5980;
}

QPushButton#QPB_Load:disabled {
    background-color: #95a5a6;
    color: #ecf0f1;
}
</string>
    </property>
    <property name="text">
     <string>📂 Charger</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
